resources/lang/en/posts.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+return [
+    'create'        => [
+        'title' => 'New Post',
+    ],
+    'fields'        => [
+        'name'          => 'Name',
+        'parent'        => 'Parent Entity',
```

Comment:
Parent entity in this case would be wrong, when compared to the terminology in the rest of the app. I use "parent" when it's the same nested. You can remove this key, and use `crud.fields.entity` in the form instead